buildtest/cli/build.py

```diff
@@ -308,10 +308,14 @@ def discover_buildspecs_by_tags(buildspec_cache, tagnames):
     buildspecs_by_tags = {}
 
     buildspecs = []
+    multiple_taglist = []
     # query all buildspecs from BUILDSPEC_CACHE_FILE for tags keyword and
     # if it matches input_tag we add buildspec to list
-
+  
     for name in tagnames:
+        multiple_taglist.extend(name.split(","))
+
+    for name in multiple_taglist:
```

Comment:
i think if you run  `buildtest build -t ,, ` there is no need to go through the entire for loop to discover the buildspecs by tags. 

We should expect to see `tagnames=",,"` and the value of `multiple_taglist` would be an empty list if that is the case we may need to return 
That being said i think we can add a condition like this prior to doing the `for name in multiple_taglist`

```
if not multiple_taglist:
   return buildspecs_by_tags, buildspecs
```